nebula_hw_interfaces/include/nebula_hw_interfaces/nebula_hw_interfaces_hesai/hesai_hw_interface.hpp

```diff
@@ -50,77 +50,66 @@
 
 namespace nebula::drivers
 {
-const int PandarTcpCommandPort = 9347;
-const uint8_t PTC_COMMAND_DUMMY_BYTE = 0x00;
-const uint8_t PTC_COMMAND_HEADER_HIGH = 0x47;
-const uint8_t PTC_COMMAND_HEADER_LOW = 0x74;
-const uint8_t PTC_COMMAND_GET_LIDAR_CALIBRATION = 0x05;
-const uint8_t PTC_COMMAND_PTP_DIAGNOSTICS = 0x06;
-const uint8_t PTC_COMMAND_PTP_STATUS = 0x01;
-const uint8_t PTC_COMMAND_PTP_PORT_DATA_SET = 0x02;
-const uint8_t PTC_COMMAND_PTP_TIME_STATUS_NP = 0x03;
-const uint8_t PTC_COMMAND_PTP_GRANDMASTER_SETTINGS_NP = 0x04;
-const uint8_t PTC_COMMAND_GET_INVENTORY_INFO = 0x07;
-const uint8_t PTC_COMMAND_GET_CONFIG_INFO = 0x08;
-const uint8_t PTC_COMMAND_GET_LIDAR_STATUS = 0x09;
-const uint8_t PTC_COMMAND_SET_SPIN_RATE = 0x17;
-const uint8_t PTC_COMMAND_SET_SYNC_ANGLE = 0x18;
-const uint8_t PTC_COMMAND_SET_TRIGGER_METHOD = 0x1b;
-const uint8_t PTC_COMMAND_SET_STANDBY_MODE = 0x1c;
-const uint8_t PTC_COMMAND_SET_RETURN_MODE = 0x1e;
-const uint8_t PTC_COMMAND_SET_CLOCK_SOURCE = 0x1f;
-const uint8_t PTC_COMMAND_SET_DESTINATION_IP = 0x20;
-const uint8_t PTC_COMMAND_SET_CONTROL_PORT = 0x21;
-const uint8_t PTC_COMMAND_SET_LIDAR_RANGE = 0x22;
-const uint8_t PTC_COMMAND_GET_LIDAR_RANGE = 0x23;
-const uint8_t PTC_COMMAND_SET_PTP_CONFIG = 0x24;
-const uint8_t PTC_COMMAND_GET_PTP_CONFIG = 0x26;
-const uint8_t PTC_COMMAND_RESET = 0x25;
-const uint8_t PTC_COMMAND_SET_ROTATE_DIRECTION = 0x2a;
-const uint8_t PTC_COMMAND_LIDAR_MONITOR = 0x27;
-
-const uint8_t PTC_ERROR_CODE_NO_ERROR = 0x00;
-const uint8_t PTC_ERROR_CODE_INVALID_INPUT_PARAM = 0x01;
-const uint8_t PTC_ERROR_CODE_SERVER_CONN_FAILED = 0x02;
-const uint8_t PTC_ERROR_CODE_INVALID_DATA = 0x03;
-const uint8_t PTC_ERROR_CODE_OUT_OF_MEMORY = 0x04;
-const uint8_t PTC_ERROR_CODE_UNSUPPORTED_CMD = 0x05;
-const uint8_t PTC_ERROR_CODE_FPGA_COMM_FAILED = 0x06;
-const uint8_t PTC_ERROR_CODE_OTHER = 0x07;
-
-const uint8_t TCP_ERROR_UNRELATED_RESPONSE = 1;
-const uint8_t TCP_ERROR_UNEXPECTED_PAYLOAD = 2;
-const uint8_t TCP_ERROR_TIMEOUT = 4;
-const uint8_t TCP_ERROR_INCOMPLETE_RESPONSE = 8;
-
-const uint16_t PANDARQT64_PACKET_SIZE = 1072;
-const uint16_t PANDARQT128_PACKET_SIZE = 1127;
-const uint16_t PANDARXT32_PACKET_SIZE = 1080;
-const uint16_t PANDARXT32M_PACKET_SIZE = 820;
-const uint16_t PANDARAT128_PACKET_SIZE = 1118;
-const uint16_t PANDAR64_PACKET_SIZE = 1194;
-const uint16_t PANDAR64_EXTENDED_PACKET_SIZE = 1198;
-const uint16_t PANDAR40_PACKET_SIZE = 1262;
-const uint16_t PANDAR40P_EXTENDED_PACKET_SIZE = 1266;
-const uint16_t PANDAR128_E4X_PACKET_SIZE = 861;
-const uint16_t PANDAR128_E4X_EXTENDED_PACKET_SIZE = 1117;
-const uint16_t MTU_SIZE = 1500;
+const int g_pandar_tcp_command_port = 9347;
```

Comment:
In some of my code I am using lots of these as well and was wondering which guidelines did you follow for the `g_` prefix. Additionally, would constexpr be better here?

Reply:
This one is (regrettably) part of Autoware's `.clang-tidy` configuration, which we based ours on:
https://github.com/autowarefoundation/autoware/blob/0e44b14187196ab0fd75a69bf0eee27b086d3b47/.clang-tidy#L447-L448

Also not a fan but I think it's best to keep the projects mostly in sync.